### DISCUSSION TITLE
Set_real_ip_from and IPv6 does not work

### DISCUSSION THREAD

**Discussion by Emporea** [nginx-proxy/nginx-proxy#1899] [Q&A]
Architecture in this example.

`Internet(IPv6) -> Nginx-Proxy -> example_container`

IPv4 works fine, and the public ip of the client is forwarded to example_container
But IPv6 returns e.g. 172.19.0.1 no matter the client real ip.
This is the docker network bridge gateway.

Nginx-Proxy listens on both 80 / 443 and Ipv4 / Ipv6
`0.0.0.0:80->80/tcp, :::80->80/tcp, 0.0.0.0:443->443/tcp, :::443->443/tcp`

How can I forward the real ipv6?

**Comment by buchdag** [comment]
This might help : https://github.com/nginx-proxy/nginx-proxy/issues/1419#issue-594554477
